docs/index.md

```diff
@@ -0,0 +1,108 @@
+# OpenFF NAGL
+
+A playground for applying graph convolutional networks to molecules, with a focus on learning continuous "atom-type" embeddings and from these classical molecule force field parameters.
+
+## Getting started
+
+OpenFF recommends using Conda virtual environments for all scientific Python work. NAGL can be installed into a new Conda environment named `nagl` with the [`openff-nagl`] package:
+
+```shell
+mamba create -n nagl -c conda-forge openff-nagl
+conda activate nagl
+```
+
+For more information on installing NAGL, see [](installation.md).
+
+NAGL can then be imported from the [`openff.nagl`] module:
+
+```python
+import openff.nagl
+```
+
+Or executed from the command line:
+
+```shell
+openff-nagl --help
+```
+
+[`openff-nagl`]: https://anaconda.org/conda-forge/openff-nagl
+[`openff.nagl`]: openff.nagl
+
+## Inference with NAGL
+
+NAGL GNN models are used via the [`openff.nagl.GNNModel`] class. A checkpoint file produced by NAGL can be loaded with the [`GNNModel.load_from_checkpoint()`] method:
+
+```python
+from openff.nagl import GNNModel
+
+model = GNNModel.load_from_checkpoint("trained_model.chkpt")
+```
```

Comment:
Are checkpoint files intended to be the way that models are distributed long-term? And what's the plan for making these files available to users? Probably a question for @lilyminium

Reply:
We had a discussion about this live -- I summarised some notes here: https://github.com/openforcefield/openff-nagl/pull/26#discussion_r1120953811